src/OrchardCore.Modules/OrchardCore.ContentTypes/Editors/ContentTypeIndexingSettingsDisplayDriver.cs

```diff
@@ -0,0 +1,35 @@
+using System.Threading.Tasks;
+using OrchardCore.ContentManagement.Metadata.Models;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.ContentTypes.ViewModels;
+using OrchardCore.DisplayManagement.Views;
+
+namespace OrchardCore.ContentTypes.Editors
+{
+    public class ContentTypeIndexingSettingsDisplayDriver : ContentTypeDefinitionDisplayDriver
+    {
+        public override IDisplayResult Edit(ContentTypeDefinition contentTypeDefinition)
+        {
+            return Initialize<ContentTypeIndexingSettingsViewModel>("ContentTypeIndexingSettings_Edit", model =>
+            {
+                var settings = contentTypeDefinition.GetSettings<ContentTypeIndexingSettings>();
+
+                model.IsFullTextLiquid = settings.IsFullText;
+                model.FullTextLiquid = settings.FullText;
+            }).Location("Content:6");
+        }
+
+        public override async Task<IDisplayResult> UpdateAsync(ContentTypeDefinition contentTypeDefinition, UpdateTypeEditorContext context)
+        {
+            var model = new ContentTypeIndexingSettingsViewModel();
+
+            if (await context.Updater.TryUpdateModelAsync(model, Prefix))
+            {
+                context.Builder.IsFullText(model.IsFullTextLiquid);
+                context.Builder.FullText(model.FullTextLiquid);
+            }
+
```

Comment:
The only thing I would suggest is we probably don't need those extensions.
The pattern seems a little different all over the place, but I think the way it's been done in the `AutoroutePartSettingsDisplayDriver` is probably the best way forward, but up to you.

```
                context.Builder.WithSettings(new AutoroutePartSettings
                {
                    Pattern = model.Pattern,
                    AllowCustomPath = model.AllowCustomPath,
                    AllowUpdatePath = model.AllowUpdatePath,
                    ShowHomepageOption = model.ShowHomepageOption
                });
```
 just saw this template manager validation as well in autoroute, this is probably worth doing. We should do it elsewhere as well

if (!string.IsNullOrEmpty(model.Pattern) && !_templateManager.Validate(model.Pattern, out var errors))